### PR TITLE
add is_default action

### PR DESCRIPTION
**Pull request by YiTing623**
Add is_default field in credit_card.py, payment_method.py, paypal.py and factories.py
Add set default payment method in routes.py
Add test_set_payment_method_as_default and test_set_new_default_unsets_previous_default

### CODE REVIEW

**Comment by nedjulius** [comment]
is this ready for a review @YiTing623 ?

**Comment by YiTing623** [comment]
> is this ready for a review @YiTing623 ?

Yes, please take a look when you have a moment, thanks!

File: service/models/payment_method.py [hunk 1]
```diff
@@ -164,3 +165,8 @@ def find_by_user_id(cls, user_id, q=None):
         if q is None:
             q = cls.query
         return q.filter(cls.user_id == user_id)
+
+    @classmethod
+    def find_default_for_user(cls, user_id):
```
Comment (nedjulius) [inline service/models/payment_method.py:170]: this is not used anywhere but in the test. maybe you can move the logic from the route to this specific method, and call it something like `set_default_for_user` ?

**Review by nedjulius — COMMENTED** [review]
few suggestions. otherwise looks good

**Comment by YiTing623** [comment]
> few suggestions. otherwise looks good

updated. not sure if it is correct, please help review again, thanks!

**Review by nedjulius — APPROVED** [review]
looks good to me 👍

**Pull request merged**